.github/workflows/turbo-deploy.yml

```
@@ -0,0 +1,229 @@
+            client/products/${{ matrix.project }}/package-lock.json
+
+      - name: Install dependencies (root)
+        run: |
+          cd ../../../
+          rm -rf node_modules package-lock.json
+          npm install
+
+      - name: Install dependencies (lib)
+        run: |
+          cd ../../
+          rm -rf node_modules package-lock.json  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)